RELEASE_NOTES.rst

```diff
@@ -1,3 +1,126 @@
+- `CSVLintBear` - Checks syntax of CSV files.
+
+**Bears with New Settings:**
+
+You can look up the new settings at https://coala.io/languages.
+
+- `InvalidLinkBear`
+- `GitCommitBear`
+- `YapfBear`
+- `SCSSLintBear`
+
+**Deleted Settings:**
+
+- `CSecurityBear` does not accept a `neverignore` setting anymore. Those kinds
+  of issues are and should be consistently handled by coala.
+
+**Deprecated Settings:**
+
+- The `ignore_regex` setting from `InvalidLinkBear` was deprecated in favour of
+  `link_ignore_regex` for more clarity in coafiles.
+- `KeywordBear` accepts only one `keywords` argument. The `ci_keywords` and
+  `cs_keywords` arguments have been deprecated.
+- `JSHintBear` provides an `es_version` argument that implies the
+  `use_es6_syntax` argument. The latter has been deprecated in favour of the
+  former.
+- `JSHintBear` provides a more flexible `javascript_strictness` argument
+  instead of the `allow_global_strict` which has been deprecated.
+- `RuboCopBear` uses `naming_convention` instead of `name_case` now.
+
+**Other Bear Enhancements:**
+
+- The `coala-bears` package does no longer require Java upon installation.
+- `VultureBear` picks up global dependencies.
+- `ESLintBear` shows errors as `WARNING` in coala. This simplifies debugging
+  bad ESLint configurations.
+- `KeywordBear` can now automatically remove TODO comments for any language
+  coala has `Language` definitions for.
+- `FilenameBear` supports the `spacecase` convention.
+- `KeywordBear` checks for `todo` and `fixme` by default.
+- `GitCommitBear` has improved result messages.
+- `YAMLLintBear` does not check for `document-start` by default. This was not
+  a commonly chosen setting.
+- `YapfBear` will pass files in-memory to save precious IO time.
+
+**Major API Changes:**
+
+- The bears testing modules were moved to the coalib and are deprecated.
+
+**Bug Fixes:**
+
+- `AnnotationBear` yields a `HiddenResult` with an error message instead of
+  raising an exception, when the desired language is not available.
+- `AnnotationBear` yields correct results for escaped strings.
+  (https://github.com/coala/coala-bears/issues/993)
+- `AnnotationBear` yields correct results for rare corner cases of multiline
+  strings. (https://github.com/coala/coala-bears/issues/1006)
+- An issue where `LatexLintBear` crashed has been resolved.
+  (https://github.com/coala/coala-bears/issues/317)
+- `InvalidLinkBear` parses links within backticks properly. It also ignores
+  links with placeholders like `%s` and others.
+- `InvalidLinkBear` ignores FTP links.
+  (https://github.com/coala/coala-bears/issues/906)
+- `DartLintBear` emits an error when it cannot satisfy given settings.
+  (https://github.com/coala/coala-bears/issues/897)
+- `CheckstyleBear` emits an error when it cannot satisfy given settings.
+  (https://github.com/coala/coala-bears/issues/898)
+- An issue where `CheckstyleBear` cannot parse a preset configuration like
```

Comment:
`CheckstyleBear`: preset configurations `google` and `sun` are no longer downloaded from the Internet.